### ISSUE TITLE
NTR: life cycle stage and life cycle

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/Experimental-OBO-Core#40]
Needs generalization of this beyond animals. TBD: GO biological phase

```
id: UBERON:0000105
name: life cycle stage
def: "A spatiotemporal region encompassing some part of the life cycle of an organism." [https://orcid.org/0000-0002-6601-2165]
subset: efo_slim
subset: upper_level
synonym: "developmental stage" NARROW []
synonym: "stage" NARROW []
xref: BILS:0000105
xref: EFO:0000399
xref: FBdv:00007012
xref: FMA:24120
xref: HsapDv:0000000
xref: MmusDv:0000000
xref: ncithesaurus:Developmental_Stage
xref: OlatDv:0000010
xref: PdumDv:0000090
xref: WBls:0000002
xref: XAO:1000000
xref: ZFS:0000000
xref: ZFS:0100000
is_a: UBERON:0000000 ! processual entity
relationship: part_of UBERON:0000104 ! life cycle
property_value: editor_note "this class represents a proper part of the life cycle of an organism. The class 'life cycle' should not be placed here" xsd:string
property_value: external_ontology_notes "the WBls class 'all stages' belongs here as it is the superclass of other WBls stages" xsd:string
property_value: external_ontology_notes "we map the ZFS unknown stage here as it is logically equivalent to saying *some* life cycle stage" xsd:string
```

cc @vanaukenk @balhoff

**Comment by mellybelly** [comment]
Agree it should not be specific to animals. PO and plants in general have perhaps more spatial granularity in their stages (e.g. plant organ stages), but this is not specific to plants in any way, we similarly have heart organ stages, for example.

I think i agree RE occurrent vs. process; some life stages are process boundaries - for example, neural tube closure or hatching (though biology does not seem to do things in zero dimensions/instantaneously, not sure how this fits with BFO strictness).

**Comment by cmungall** [comment]
See also related ticket for biological phases, e.g. cell cycle stage: #51

**Comment by cmungall** [comment]
Tree rooted in occurrent:

* [] owl:Thing
    * [i] BFO:0000001 ! entity
        * [i] **BFO:0000003 ! occurrent**
            * [i] BFO:0000035 ! process boundary
            * [i] BFO:0000015 ! process
                * [i] BFO:0000182 ! history
                * [i] BFO:0000144 ! process profile
            * [i] BFO:0000011 ! spatiotemporal region
            * [i] BFO:0000008 ! temporal region
                * [i] BFO:0000148 ! zero-dimensional temporal region
                * [i] BFO:0000038 ! one-dimensional temporal region

(this is the OBO version of BFO - I believe there is another more up to date one that doesn't have process profile, but that is not released to the OBO PURL)

Even if we do not import all of BFO we should be consistent. I have lost the ability to mentally reason over the BFO elucidations. However I think it seems reasonable to restrict processes to things that biologists naturally feel are processes or events, and to have history/phase be a sibling

**Comment by wdduncan** [comment]
Maybe part of the confusion stems from changes between BFO-1.1, BFO-2.0, and BFO-2020. 

In BFO-1.1, `process` was a subtype of `processual_entity`. `processual_enity` was defined:
```
An occurrent [span:Occurrent] that exists in time by occurring or happening, has temporal parts and always involves and depends on some entity.
```
and `process` was defined as:
```
A processual entity [span:ProcessualEntity] that is a maximally connected spatiotemporal whole and has bona fide beginnings and endings corresponding to real discontinuities.
```
(BFO-1.1)
<img width="222" alt="image" src="https://user-images.githubusercontent.com/3186638/186720728-d84eefb8-d3fb-45ac-9f5f-316f5402da72.png">

In BFO-2.0 (classes-only version), `processual_entity` was replaced by `process`, and `process` was defined as:
```
p is a process = Def. p is an occurrent that has temporal proper parts and for some time t, p s-depends_on some material entity at t. (axiom label in BFO2 Reference: [083-003])
```
(BFO-2.0)
<img width="212" alt="image" src="https://user-images.githubusercontent.com/3186638/186720996-74591939-5ca8-4201-a87c-777d5b1ef839.png">

In [BFO-2020](https://raw.githubusercontent.com/BFO-ontology/BFO-2020/master/21838-2/owl/bfo-2020.ofn), `process profile` was removed, with `process` defined as:
```
(Elucidation) p is a process means p is an occurrent that has some temporal proper part and for some time t, p has some material entity as participant at t
```
(BFO-2020)
<img width="215" alt="image" src="https://user-images.githubusercontent.com/3186638/186721186-025cb8f9-6185-4dad-9911-aff796b222ee.png">

Both BFO-1.1 and BFO-2.0 require that processes depend on some entity (material entities in the case of BFO-1.1). BFO-2020 is weaker in this commitment.

From the discuss so far, it seems to me that life stages are dependent on the organisms that have such stages, and, thus, more appropriately represented as some kind of process. However, it is unclear which BFO "process" should be used.

**Comment by nataled** [comment]
> From the discuss so far, it seems to me that life stages are dependent on the organisms that have such stages, and, thus, more appropriately represented as some kind of process. However, it is unclear which BFO "process" should be used.

Something seems off to me about this line of reasoning. It seems based on the idea that all occurrents with material entity participants are processes; that is, as if 'process' has the logical definition (occurrent and has_participant some material entity). Is this the case? I thought the definition (in intention, if not precisely stated this way) was more like a process is a subclass_of(occurrent) and subclass_of(has_participant some material entity). Apologies for pseudo-code shorthand.

**Comment by wdduncan** [comment]
>  I thought the definition (in intention, if not precisely stated this way) was more like a process is a subclass_of(occurrent) and subclass_of(has_participant some material entity)

Yes. I think BFO-2020 is weaker. Roughly, processes are occurrents that have material entities as participants. The owl version of BFO-2020 doesn't have an axiom asserting the participation of material entities, though.

<img width="292" alt="image" src="https://user-images.githubusercontent.com/3186638/186741247-9db89f7e-2db3-4c7f-b273-7c055cfc5bf1.png">

**Comment by alanruttenberg** [comment]
BTW, you should always look at the FOL when there are questions as the OWL is only an approximation to the FOL has not been elaborated to the extent possible.

**Comment by wdduncan** [comment]
@alanruttenberg in the [participatation cl](https://github.com/BFO-ontology/BFO-2020/blob/master/21838-2/common-logic/participation.cl) file, I see the [axiom](https://github.com/BFO-ontology/BFO-2020/blob/8792dde1f5702f291adc1c4c8ff83cff28d91be1/21838-2/common-logic/participation.cl#L17):
```
(cl:comment "At every time a process exists it has a participant [trl-1]"
    (forall (p t)
     (if (instance-of p process t) (exists (c) (participates-in c p t)))))
```
I did not find an axiom asserting that the participant had to be a material entity. So, I suppose the FOL and the elucidation differ in this respect?

**Comment by alanruttenberg** [comment]
Well, there you have it. 
Good thing we have some axioms to set the record straight ;-)

**Comment by alanruttenberg** [comment]
The domain of the participates in relations are sdc, gdc, or ic but not spatial region. Together with the axioms that there must be a participant and that sdc and gdc participants imply a bearer participates, and bearers are material, one would conclude that it must be a material entity. When I'm next doing that sort of thing, I'll try to show it's provable. Could you submit an issue to the BFO-2020 issue tracker so it won't be forgotten?

**Comment by wdduncan** [comment]
> Together with the axioms that there must be a participant and that sdc and gdc participants imply a bearer participates

Yes. Thanks for pointing this out! I believe these FOL axioms entail this: [SDC](https://github.com/BFO-ontology/BFO-2020/blob/8792dde1f5702f291adc1c4c8ff83cff28d91be1/21838-2/common-logic/participation.cl#L40), [GDC](https://github.com/BFO-ontology/BFO-2020/blob/8792dde1f5702f291adc1c4c8ff83cff28d91be1/21838-2/common-logic/participation.cl#L52). It would be nice if this was reflected in a property chain in BFO-2020.  
I'm not seeing the requirement that the IC must be a material entity, though. So, I'm wondering if the elucidation is too strong.  

> Could you submit an issue to the BFO-2020 issue tracker so it won't be forgotten?

Happy to do so :)

Does any of this discussion shed light on the issue concerning life cycle stages/cycles?

**Comment by matentzn** [comment]
With all this great discussion here in mind, in Uberon, we are adding "life cycle stage" under BFO:process, see https://github.com/obophenotype/uberon/issues/2301.

Once this is done I would like to suggest to add these two classes to COB so we can align all other anatomy ontologies using the same pattern. We can continue to refine the modelling over time.

Is there anyone _against_ adding both "life cycle stage" and "life cycle" to COB and anyone that will mutiny if we put both under BFO:process?
